hledger-core.el

```diff
@@ -53,8 +53,14 @@
   "Regular expression for dates for font lock.")
 (defvar hledger-date-and-desc-regex (format "\\<%s\\s-*\\*?\\s-*[^[:space:]]+\\>" hledger-date-regex)
   "Regular expression for matching a starting entry with some description.")
-(defvar hledger-account-regex "\\(\\([Rr]evenues\\|[aA]ssets\\|[lL]iabilities\\|[Ee]quity\\|[Ee]xpenses\\|[iI]ncome\\|[Zz]adjustments\\)\\(:[A-Za-z--]+\\)+\\)"
-  "Regular expression for a potential journal account.")
+
+(defvar hledger-account-regex nil "Regular expression for a potential journal account.")
```

Comment:
Thanks for the feedback @narendraj9! I have just changed it so that we are setting the value at the same time as we are defining the variable - I don't think it's as important to change, at least for now. I have also made some changes so that the tests are extracted out of the main codebase. I followed some conventions that I found in other projects, which seem to use Cask in order to facilitate running tests from the command line. Please let me know what you think. I will follow up soon with an email to talk about maintainership. Thanks again and talk to you soon!